app/views/smart_answers/inputs/_money_question.html.erb

```diff
@@ -9,7 +9,7 @@
   heading_size: "l",
   width: 10,
   hint: question.hint_text,
-  suffix: question.suffix_label,
+  suffix: (question.suffix_label if question.suffix_label.present?),
```

Comment:
Good catch! Think if we put this logic into the question presenter https://github.com/alphagov/smart-answers/blob/68dd25abae9f324ca9627b5fa7be8b31d95e746b/app/presenters/question_presenter.rb#L48, we can test it too.

Reply:
That makes sense - thanks. I've moved the modification to the presenter.